packages/expo-calendar/CHANGELOG.md

```diff
@@ -10,9 +10,12 @@
 
 ### 💡 Others
 
-## 12.2.1 — 2023-12-19
+## 12.2.1 — 2023-12-22
+
+### 🎉 New features
+
+- Standardize `getEventsAsync` to return events sorted by start date ([#28353](https://github.com/expo/expo/pull/28353) by [@demfabris](https://github.com/demfabris))
```

Comment:
That entry should be under `Unpublished > Bug fixes` section above. The version `12.2.1`   was already published.